docs/_layouts/default.html

```diff
@@ -3,16 +3,36 @@
 
 <head>
   <title>{{ page.title }}</title>
-  <!-- link to main stylesheet -->
-  <link rel="stylesheet" type="text/css" href={{ "/assets/css/main.css" | relative_url }}>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+   <!-- Bootstrap CSS -->
+   <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
```

Comment:
Opted to use CDN than downloading the File to the repo. I don't know if we need to use npm package manager or a possibility of that with github pages. Need more research if we want to package manager.

Reply:
I'm okay using the CDN in production if we're not concerned about there being outages.

Reply:
We are using Github for hosting for documentation and outage may happen from that as well. But I think i won't overthink for now at-least.